torch/_inductor/compile_fx.py

```diff
@@ -70,12 +70,24 @@
 from .virtualized import V
 
 if config.is_fbcode():
+    from aiplatform.runtime_environment.runtime_environment_pybind import (
+        RuntimeEnvironment,
+    )
     from torch._inductor.fb.utils import time_and_log
 else:
     # no-op decorator
     def time_and_log(attr: str, extra_loggings: Optional[Dict[str, str]] = None):
         return dynamo_utils.identity
 
+    class RuntimeEnvironment:
```

Comment:
it will log None to cols by default so we don't need a fallback here.

Reply:
I see. Thanks for the comments.